prover_server/prover_server.cpp

```diff
@@ -282,36 +278,31 @@ static void RunServer(
 
     // Finally assemble the server.
     std::unique_ptr<grpc::Server> server(builder.BuildAndStart());
-    std::cout << "[DEBUG] Server listening on " << server_address << std::endl;
+    std::cout << "[INFO] Server listening on " << server_address << "\n";
 
     // Wait for the server to shutdown. Note that some other thread must be
     // responsible for shutting down the server for this call to ever return.
     display_server_start_message();
     server->Wait();
 }
 
-#ifdef ZETH_SNARK_GROTH16
-static snark::keypair load_keypair(const std::string &keypair_file)
-{
-    std::ifstream in(keypair_file, std::ios_base::in | std::ios_base::binary);
-    in.exceptions(
-        std::ios_base::eofbit | std::ios_base::badbit | std::ios_base::failbit);
-    return snark::keypair_read_bytes(in);
-}
-#endif
-
 int main(int argc, char **argv)
 {
     // Options
     po::options_description options("");
     options.add_options()(
-        "keypair,k", po::value<std::string>(), "file to load keypair from");
-#ifdef DEBUG
+        "keypair,k",
+        po::value<std::string>(),
+        "file to load keypair from. If it doesn't exist, a new keypair will be "
+        "generated and written to this file. (defaults to [");
```

Comment:
The default should be $HOME assuming we run on a POSIX compliant OS

Reply:
Updated the help.  Thanks.